releasenotes/notes/fix-target-instr-supported-900a1caa76e30655.yaml

```diff
@@ -2,5 +2,5 @@
 fixes:
   - |
     Fixed a bug in the :meth:`.Target.instruction_supported` method where 
-    targets with ``self.num_qubits==None`` would always return ``false`` 
-    independently of the  supported basis set. 
\ No newline at end of file
+    targets with ``self.num_qubits=None`` would always return ``False``
```

Comment:
Tiny nitpick but I'd say `==` was more correct here; in my head that translates to "if the number of qubits is None", whereas `=` translates to "if you set the number of qubits to None" (and you can't set the `Target.num_qubits` like this I think 😛).

Reply:
Sure, changing that!